x-pack/test/security_solution_cypress/cypress/e2e/entity_analytics/entity_analytics_management_page.cy.ts

```diff
@@ -147,3 +156,33 @@ describe(
     });
   }
 );
+
+// this test suite doesn't run on serverless because it requires a custom role
+describe(
+  'Entity analytics management page - Risk Engine Privileges Callout',
+  { tags: ['@ess'] },
+  () => {
+    it('should not show the callout for superuser', () => {
+      cy.intercept(RISK_ENGINE_PRIVILEGES_URL).as('getPrivileges');
+      login();
+      visit(ENTITY_ANALYTICS_MANAGEMENT_URL);
+      cy.wait('@getPrivileges', { timeout: 15000 });
+      cy.get(RISK_SCORE_STATUS_LOADING, { timeout: 2000 }).should('not.exist');
```

Comment:
This timeout can cause flaky tests on CI.

Reply:
@MadameSheema can you provide more details about why this causes flakiness? Is the recommendation to remove the timeout?

Reply:
I reduced the timeout because I am using a 'not.exist', leaving the default means cypress will wait 4 seconds just to make sure the element doesn't turn up in that time, I will remove them to keep it simpler 👍